src/java/org/apache/cassandra/index/sai/disk/v1/V1SearchableIndex.java

```diff
@@ -169,6 +175,12 @@ public List<RangeIterator<Long>> search(Expression expression,
     {
         List<RangeIterator<Long>> iterators = new ArrayList<>();
 
+        if(keyRange instanceof Bounds && keyRange.left.equals(keyRange.right) && keyRange.left instanceof DecoratedKey)
```

Comment:
nit: code style format.
```suggestion
        if (keyRange instanceof Bounds && keyRange.left.equals(keyRange.right) && keyRange.left instanceof DecoratedKey)
```

Reply:
and other places as well..